modules/core/src/main/java/org/apache/ignite/internal/IgniteFeatures.java

```diff
@@ -194,7 +194,10 @@
     CACHE_GROUP_KEY_CHANGE(50),
 
     /** Possibility to safe deactivation, take into account pure in memory caches with possible data loss.*/
-    SAFE_CLUSTER_DEACTIVATION(51);
+    SAFE_CLUSTER_DEACTIVATION(51),
+
+    /** Statistics collection. */
+    STATISTICS_COLLECTION(52);
```

Comment:
BTW id=52 already reserved. Please check https://ggsystems.atlassian.net/wiki/spaces/GG/pages/1192198276/Community+edition+features+list for next feature ID. You need to reserve it first.